.travis/test.sh

```diff
@@ -63,7 +36,7 @@ catkin_make ${CM_OPTIONS} \
   || (gh-pr-comment "${build_number} FAILED on ${ROS_DISTRO}" '```catkin_make``` failed'; false)
 catkin_make tests ${CM_OPTIONS} \
   || (gh-pr-comment "${build_number} FAILED on ${ROS_DISTRO}" '```catkin_make tests``` failed'; false)
-catkin_make run_tests ${CM_OPTIONS} --make-args -j1 -l1 \
+catkin_make run_tests ${CM_OPTIONS} \
```

Comment:
seems not yet stable enough to run concurrently on Travis
```suggestion
catkin_make run_tests ${CM_OPTIONS} --make-args -j1 -l1 \
```